test/integration/exchanged_token_integration_test.cc

```diff
@@ -220,9 +220,10 @@ class ExchangedTokenIntegrationTest : public HttpProtocolIntegrationTest {
   }
 };
 
-INSTANTIATE_TEST_SUITE_P(Protocols, ExchangedTokenIntegrationTest,
-                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams()),
-                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+INSTANTIATE_TEST_SUITE_P(
+    Protocols, ExchangedTokenIntegrationTest,
+    testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParamsWithoutHTTP3()),
```

Comment:
I don't think we should exclude h3?

Reply:
The issue is that requests have ":authority" set to "host", so if you change them there, it'll pass with h3.

Reply:
Looks like the tests were passing (maybe an asan flake). Will try reverting the withoutH3 to see how that does,